### DISCUSSION TITLE
W10: W7 classic style start menu doesn't scale properly

### DISCUSSION THREAD

**Discussion by Ledragonnantais** [Open-Shell/Open-Shell-Menu#1806] [Q&A]
Hello OpenShell community. I have a question as I just installed W10 a week and half ago and get my hands into modding it to make it look like W7. It's almost a success, but I have a problem with the start menu. In fact, this last "doesn't scale" properly. Compared to my start menu on my real W7, the one in W10 is taller and I have a massive blank space between "executer" (run) and turn-OFF button, as you can see in the joined picture (with red bar and green bar). 
Is there a way in the options, a value to change, a case to tick, to make it look like W7 start menu with the same option(my doc, music, images, download, computer, control panel, devices and printer, run), but small in height, and without that ugly blank space?
Thanks a lot, have a nice day ^^
![start menu](https://github.com/Open-Shell/Open-Shell-Menu/assets/59211240/e58eb5ef-31c7-4684-b181-268fc66a7e3e)

**Comment by CTVCAM8** [comment]
![Adjust menu height](https://github.com/Open-Shell/Open-Shell-Menu/assets/44127216/99eb55c7-3f28-47c5-9197-6059ac0a5b2c)


Try playing with the Main Menu height.

**Reply by Ledragonnantais** [reply]
Hello CTV. Thanks a ton for the reply. That helped a little better. It's not as perfect as true W7 menu, but it reduced greatly the space. ^^
